packages/react-html/README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt) [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-html.svg)](https://badge.fury.io/js/%40shopify%2Freact-html)
 
-A component to render your React app with no static HTML.
+A collection of utilities for constructing an HTML document that supports serialization, assets, and various `head` tags.
```

Comment:
I like the taglines to be a bit briefer / punchier personally.
> React components for server rendering HTML documents and making serializing data between the client and server scalable.

Maybe even
> Say goodbye to react-tree-walker and JSON.parse.

Reply:
> React components for server rendering HTML documents and making serializing data between the client and server scalable.

❤️ 

> Say goodbye to react-tree-walker and JSON.parse.

That one would raise the bar quite a bit (requires prior knowledge), but maybe it could be used as additional context for experts? For example:

> React components for server rendering HTML documents and making serializing data between the client and server scalable. Useful as a replacement for [react-tree-walker](https://github.com/ctrlplusb/react-tree-walker) and `JSON.parse`.

Reply:
`react-extract` uses `react-tree-walker` under the hood so I don't think it's appropriate to say goodbye to it. I would prefer to leave it as-is since we will soon add title/ meta/ link tag support here, which make focusing on serialization a bit too limited. Happy to punch it up if we have an alternative that makes it clear this is intended for generally managing an HTML document.

Reply:
How about splitting it up a bit:

> A collection of utilities for constructing an HTML document.

Features:
* Serialization
* Assets
* `head` tags